.circleci/config.yml

```diff
@@ -4,7 +4,7 @@ aliases:
   - &default-setup
     working_directory: *working-directory
     docker:
-      - image: 'circleci/node:9.10'
+      - image: 'circleci/node:8.16'
```

Comment:
This is to resolve circleci setup, `get-caller-file` requiring`package.json/engines/node` to be `"node": "6.* || 8.* || >= 10.*"`.